spec/GameSpec.js

```diff
@@ -0,0 +1,32 @@
+describe("Game", function() {
+
+  var game, allTurns;
+
+  beforeEach(function() {
+    game = new Game();
+  });
+
+  describe("default settings", function() {
+
+    it("should start with a score of 0", function() {
+      expect(game.scoreDefault).toEqual(0);
+    });
+
+  });
+
+  describe("playing a game", function() {
+
+    it("should score zero if all rolls hit no pins", function() {
+      allTurns(20, 0);
```

Comment:
allTurns is more of a helper function for your specs, not really a function of the game itself